<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkerWindow</class>
 <widget class="QDialog" name="WorkerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Worker Window</string>
  </property>
  <property name="styleSheet">
   <string>
    QWidget {
        background-color: #2E2E2E;
        font-family: Arial, sans-serif;
        font-size: 12pt;
        color: white;
    }
    QPushButton {
        background-color: #4CAF50;
        color: white;
        padding: 10px;
        border-radius: 5px;
        border: none;
        font-size: 14px;
    }
    QPushButton:hover {
        background-color: #45a049;
    }
    QLineEdit, QTextEdit {
        border: 1px solid #555;
        border-radius: 5px;
        padding: 8px;
        font-size: 12pt;
        background-color: #333;
        color: white;
    }
    QTabWidget {
        background-color: #3C3F41;
        border: 1px solid #555;
        color: white;
    }
    QTabBar::tab {
        background-color: #3C3F41;
        border: 1px solid #555;
        padding: 5px;
    }
    QTabBar::tab:selected {
        background-color: #4CAF50;
        border: 1px solid #4CAF50;
    }
    QListWidget {
        background-color: #333;
        color: white;
        border: 1px solid #555;
    }
    QListWidget::item:selected {
        background-color: #4CAF50;
        color: white;
    }
   </string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTabWidget" name="WorkerTabs">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="TabOrders">
    <attribute name="title">
     <string>Orders</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>711</width>
       <height>451</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QPushButton" name="btnLoadOrders">
        <property name="text">
         <string>Загрузити замовлення</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnTakeOrder">
        <property name="text">
         <string>Взяти замовлення</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="ordersListWidget"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="TabWorkDetails">
    <attribute name="title">
     <string>Work Details</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>681</width>
       <height>401</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QLineEdit" name="leMediaLink">
        <property name="text">
         <string>Посилання на медіадані</string>
        </property>
        <property name="placeholderText">
         <string>Media Link</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="leOrderId">
        <property name="text">
         <string>ID замовлення</string>
        </property>
        <property name="placeholderText">
         <string>Order ID</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QTextEdit" name="teWorkDescription">
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial','sans-serif'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Опис виконаної роботи&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>Work Description</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QListWidget" name="acceptedOrdersListWidget"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnLoadActiveOrders">
        <property name="text">
         <string>Показати замовлення</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btnSaveDetails">
        <property name="text">
         <string>Зберегти деталі</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="TabDoneOrders">
    <attribute name="title">
     <string>Done Orders</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>621</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QPushButton" name="btnLoadCompletedOrders">
        <property name="text">
         <string>Всі виконані замовлення</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QListWidget" name="completedOrdersListWidget"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnSearchOrders">
        <property name="text">
         <string>Пошук</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="leSearchWorker">
        <property name="text">
         <string>Введіть дані для пошуку</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
